OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -467,7 +467,12 @@ Action InitInputPanel(Widget panel)
 				var hotkeys = new Dictionary<string, string>()
 				{
 					{ "ObserverCombinedView", "All Players" },
-					{ "ObserverWorldView", "Disable Shroud" }
+					{ "ObserverWorldView", "Disable Shroud" },
+					{ "PauseKey", "Pause/Play" },
```

Comment:
I was wondering about the same thing, but then I thought Pause/Play key has more priority. I might change that, will see...